test/functional/specs/Personalization/C782719.js

```diff
@@ -83,47 +74,70 @@ test.skip("Test C782719: SPA support with auto-rendering disabled", async () =>
   }).getPayloadsByType("personalization:decisions");
 
   await t.expect(personalizationPayload.length).eql(3);
+
+  const assertRenderAttemptedFlag = resultingObject.propositions.every(
```

Comment:
The name `assertRenderAttemptedFlag` makes me thing it's a function that will assert something about the renderAttemptedFlag. It might be clearer if it were named something like `allPropositionsWereRendered`.

Reply:
yes, agree! thank you